src/bundle/Resources/public/js/scripts/core/toggle.button.js

```diff
@@ -57,10 +60,19 @@
             toggler.classList.remove('ibexa-toggle--is-focused');
         }
 
+        toggleStateOnSpacePressed(event) {
```

Comment:
What about enter key?
It should not work?

Reply:
Nope, enter does not trigger checkbox toggle